src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle electricity generation"
+    
+    SubClassOf: 
+        OEO_00050001,
+        OEO_00010234 some OEO_00000139,
+        OEO_00010234 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00050020,
+        <http://purl.obolibrary.org/obo/RO_0002427> some OEO_00310027
+    
+    
+Class: OEO_00310027
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A gas turbine process is an energy transformation that converts chemical energy into rotational energy using a continous internal combustion process. A gas turbine is participating in a gas turbine process.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "gas turbine process"
+    
+    SubClassOf: 
+        OEO_00020003,
+        OEO_00000532 some OEO_00000099,
+        OEO_00010234 some OEO_00000007,
+        OEO_00010235 some OEO_00240000,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00140038,
+        <http://purl.obolibrary.org/obo/RO_0000057> some OEO_00000185
+    
+    
+Class: OEO_00310028
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A steam power unit is a power generating unit using steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "steam power unit"
+    
+    SubClassOf: 
+        OEO_00000334,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007,
+        <http://purl.obolibrary.org/obo/BFO_0000051> some OEO_00000396
+    
+    
+Class: OEO_00310029
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A combined cycle power plant (CCGT) is a gas power plant that has gas fired power units, steam power units, gas power units and heat recovery steam units as parts.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "CCGT",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "combined cycle power plant"
+    
+    SubClassOf: 
+        OEO_00000184,
```

Comment:
Seems unnecessary to be classified as a subclass of gas power plant.  Might lead to confusion down the line when someone wants to indicated only gas turbine plants.

Reply:
I will also add the axiom `has part` some `gas fired power unit` (see comment below).

Reply:
> Makes sense, should I move it as a subclass of `power plant`?

#750 says nothing specific about "gas power plant", so I guess it got added as one variation among others.  
In "The Real World&trade;" there are "gas turbine + generator" (open cycle) plants and "gas turbine + steam turbine + generator" plants (closed cycle; this).  I guess whether there is a super class between the two and "power plant" is question of taste.
"Gas power plant" as defined in the ontology currently fits both (as both have a gas turbine), so there is no specific class for open cycle plants.  
- Either add that and make both a subclass of "gas power plant", 
- or differentiate "gas power plant" from CCGT such that it is a separate class and put both under "power plant".

Reply:
A gas turbine and and a hence a CCGT not necessarily has to be fueled with a gas, but can often be fueled with heating oil. So a `combined cycle power plant` should be a direct subclass of `power plant`.

Reply:
Good point.

Reply:
I propose a new definition:
A combined cycle power plant (CCGT) is a power plant that has a gas turbine, a steam turbine, gas fired power units and heat recovery steam units as parts.